torch/csrc/jit/mobile/runtime_compatibility.cpp

```diff
@@ -8,5 +10,45 @@ uint64_t _get_runtime_bytecode_version() {
   return caffe2::serialize::kProducedBytecodeVersion;
 }
 
+/*
+ * Returns all registered PyTorch ops and their versioning
+ */
+std::unordered_map<std::string, OperatorInfo> _get_runtime_ops_and_info() {
+  std::unordered_map<std::string, OperatorInfo> result;
+
+  // Grab the jit operators
+  auto nonDispatcherOperators = torch::jit::getAllOperators();
+  for (const auto& full_op : nonDispatcherOperators) {
+    auto op = full_op->schema();
+    int num_schema_args = op.arguments().size();
+    if (op.overload_name() != "") {
+      result.emplace(
+          op.name() + "." + op.overload_name(), OperatorInfo{num_schema_args});
+    } else {
+      result.emplace(op.name(), OperatorInfo{num_schema_args});
+    }
+  }
+
+  // Grab the dispatcher operators
+  auto dispatcherOperators = c10::Dispatcher::singleton().getAllOpNames();
+  for (auto& op : dispatcherOperators) {
+    // grab schema
+    const auto op_handle = c10::Dispatcher::singleton().findOp(op);
+    int num_schema_args = NO_SCHEMA;
+    if (op_handle->hasSchema()) {
+      num_schema_args = op_handle->schema().arguments().size();
+    }
+
+    if (op.overload_name != "") {
+      result.emplace(
+          op.name + "." + op.overload_name, OperatorInfo{num_schema_args});
+    } else {
+      result.emplace(op.name, OperatorInfo{num_schema_args});
+    }
```

Comment:
Same suggestion as above. Maybe make a little lambda function for the name?

    auto fullOpName = [](auto& op) 
    { 
        return op.overload_name() == "" ? op.name() : op.name() + "." + op.overload_name();
    };

Reply:
I think it's probably more efficient to use empty function than string comparison.
```
auto fullOpName = [](auto& op) 
{ 
    return op.overload_name().empty() ? op.name() : op.name() + "." + op.overload_name();
};
```

Reply:
I dont think I can do a lambda cleanly because what type 'op' is differs between the 2 loops. In one of them I have to go through getter functions .name() and the other I have to access the variable directly .name